utils/src/main/scala/it/pagopa/interop/commons/utils/package.scala

```diff
@@ -30,4 +31,20 @@ package object utils {
     } yield (bearerToken, correlationId, ip)
   }
 
+  type BearerToken   = String
+  type CorrelationId = String
+  type IpAddress     = String
```

Comment:
for consistency, these should be used also in the above `extractHeaders` function